### ISSUE TITLE
optionally using obs photos in the species tab rather than default taxon photos

### ISSUE THREAD

**Issue by naturalistcharlie** [inaturalist/inaturalist#1048]
I have found that when I share the neat new species tab with a list of species, with people not highly familiar with iNat, people get very confused about the photos since they only display the stock taxa photo. Sometimes I get complements on my great photos (which I usually didn't take) and other times confusion because the seasonality, other species in the photo, etc are totally wrong. while I think using those photos has its place, it would be awesome to have a filter or button to show only photos from observations within the query.

For example, here is a list of species found in Hemlock Forest, a natural community we have in Vermont:
http://www.inaturalist.org/observations?place_id=47&view=species&field:natural%20community=Hemlock%20Forest
this is a very neat page and i find i have been using it a lot but I'd prefer an option to have it show one of my observations from that particular Vermont forest rather than the stock picture of a hemlock cone, etc.

**Comment by kueda** [comment]
I agree this would be nice, though I'm not sure I'd want it to be optional. @pleary, do you think this would be feasibly performative?
